### PR TITLE
Uses best model from hyperparam opt to score on test data

### PR DESCRIPTION
**Pull request by aidanmattrick**
Based on new `analysis.py` some refactoring can be done to improve modularization of code based on new structure but wanted to add now so that we can leverage `confusion_matrix` generated.

### CODE REVIEW

File: src/predict_test.py [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+
+from lightgbm.sklearn import LGBMClassifier
+from catboost import CatBoostClassifier
+
+import altair as alt
+from vega_datasets import data
+
+from docopt import docopt
+from pylab import savefig
+from render_table import render_table
```
Comment (zmerpez) [inline src/predict_test.py]: hey there, render table is not in utils, you need to change that by 'from character_utils import*'

**Review by zmerpez — CHANGES_REQUESTED** [review]

File: src/predict_test.py [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+
+
+from lightgbm.sklearn import LGBMClassifier
+from catboost import CatBoostClassifier
+
+import altair as alt
+from vega_datasets import data
+
+from docopt import docopt
+from pylab import savefig
```
Comment (cmmclaug) [inline src/predict_test.py:35]: Do we need to add this to env?

File: src/predict_test.py [hunk 3]
```diff
@@ -0,0 +1,145 @@
...
+from render_table import render_table
+args = docopt(__doc__)
+
+def read_input_file(input_file_path):
```
Comment (cmmclaug) [inline src/predict_test.py]: This has been moved to character_utils.py now!

File: src/predict_test.py (hunk 1 above)
Comment (cmmclaug) [reply src/predict_test.py]: OR even better: from character_utils import render_table, read_input_file

**Review by cmmclaug — CHANGES_REQUESTED** [review]

File: src/predict_test.py [hunk 4]
```diff
@@ -0,0 +1,145 @@
...
+    test_df =  read_input_file(input_file_path)
+
+    print("\n\n##### Testing Model!")
+    X_test, y_test = test_df.drop(columns=['align']), test_df['align']
+
+    score = model.score(X_test, y_test)
+    print("\n\n##### Model Score:")
+    print(f'{score}')
+
+    confusion_matrix = plot_confusion_matrix(model, X_test, y_test, display_labels=["Bad", "Neutral", "Good"],
```
Comment (zmerpez) [inline src/predict_test.py]: display_labes should be in if with two different options ["bad", "good"] or all three

File: src/predict_test.py (hunk 1 above)
Comment (aidanmattrick) [reply src/predict_test.py]: Removed render_table as not being used!

File: src/predict_test.py (hunk 2 above)
Comment (aidanmattrick) [reply src/predict_test.py:35]: Using pylab for saving confusion matrix

File: src/predict_test.py (hunk 3 above)
Comment (aidanmattrick) [reply src/predict_test.py]: Using from there - thanks!

**Review by zmerpez — APPROVED** [review]
thanks

**Pull request merged**